src/briefcase/integrations/windows_sdk.py

```diff
@@ -0,0 +1,136 @@
+from importlib.util import find_spec
+from pathlib import Path
+from typing import Generator, Tuple
+
+from briefcase.exceptions import BriefcaseCommandError, MissingToolError
+from briefcase.integrations.base import Tool
+
+# winreg can only be imported on Windows
+if find_spec("winreg"):
+    import winreg
```

Comment:
The pattern we've used elsewhere is to try-except the import; and if it isn't there, set `winreg = None`. That way, you have a symbol you can reliably test that verifies if the library existed.

Reply:
Good call. Updated.